app/main.go

```diff
@@ -85,6 +87,11 @@ func (c Config) defaultHTTPClient(logger ui.Logger) *http.Client {
 
 // Main runs the Hermit command-line application with the given config.
 func Main(config Config) {
+	config.InstallPaths = []string{
```

Comment:
This shouldn't blindly overwrite this. It should check if it's empty, then use the defaults, otherwise use what the user passed in.

Reply:
Actually I don't think this should be in the global Hermit config at all, it's only used by the gen-installer command.

Reply:
This is put here to make it possible for a custom Hermit to override with their own values. Is this not desired?

Reply:
It should just be flags passed to `gen-installer`, as it was before.

Reply:
Got it.